modules/openapi-generator/src/main/resources/python/api_client.mustache

```diff
@@ -243,6 +247,26 @@ class ApiClient(object):
                                       response_data.getheaders()))
 {{/tornado}}
 
+    def parameters_to_multipart(self, params, collection_types):
+        """Get parameters as list of tuples, formatting as json if value is collection_types
+
+        :param params: Parameters as dict or list of two-tuples
```

Comment:
If we keep your code as-is, then it will always ingest tuples for params.
So if you want to keep it as-is, please change this docstring to describe params as tuples not a dict.

Reply:
@spacether ,  thank you for your feed back. Do You mean docstring should be change as bellow, collect? just confirming before update & push.
```patch
-                    :param params: Parameters as dict or list of two-tuples
+                    :param params: Parameters as list of two-tuples
```

Reply:
docstring updated by 1f74d2e31f6ea571af5fdf507bf0b331a4782c4b, please check it.

Reply:
Thank you!